dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -1,7 +1,8 @@
 from . import BaseColumnProfiler
 from .profiler_options import CategoricalOptions
 from . import utils
-
+from collections import Counter
+from collections import defaultdict
```

Comment:
Can make this 1 import line with `from collections import Counter, defaultdict`, also, this is a standard library, so its import should go above the others

Reply:
Also where are you using `Counter`? I don't see it anywhere in the code

Reply:
I believe counter is not being used so we can remove it.

Reply:
Also, need two spaces between imports and beginning of class.